packages/pangea-sdk/NLog.config

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<nlog xmlns="http://www.nlog-project.org/schemas/NLog.xsd"
+      xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+
+    <targets>
+        <target name="logfile" xsi:type="File" fileName="pangea-sdk.json" layout='{"time": "${longdate}", "name": "${logger}", "level": "${level}", "message": ${message}}'/>
```

Comment:
not sure if it's a good strategy to log in a file in the project's folder... unless the user set a debug flag of something

Reply:
user can setup whatever file they want, it's changed on the fly. This is just and example

Reply:
Not need to recompile